Create isnad.csl

mtarakci opened the pull request:
The common citation style created by Dimer (http://dimer.cumhuriyet.edu.tr) for academic journals published in the field of humanities and social sciences in Turkey. 
A meeting was held with editors of all academic journals published in theology in Turkey several months ago. At this meeting, a consensus was reached on the use of a common citation style called Isnad. Isnad was announced twenty days ago and eighteen journals have already decided to use Isnad. More than a hundred of journals or institutions in Turkey are estimated to use this citation style.
For more information, see http://www.isnadsistemi.org/
And for the style guide, see http://www.isnadsistemi.org/dosyalar/Isnad_Atif_Sistemi_12-3-2018.pdf

adam3smith commented on isnad.csl:
```diff
@@ -0,0 +1,1387 @@
+      <term name="issue" form="short">sy.</term>
+      <term name="translator" form="short">trc.</term>
+      <term name="translator" form="verb-short">trc.</term>
+      <term name="and"> - </term>
```
I don't think this is right -- you don't actually want to define "and" as a hyphen. To get something like
> Topaloğlu, Bekir - Yavuz, Y. Şevki - Çelebi, İlyas.

you'd just do

```
<names variable="author">
  <name delimiter=" - " delimiter-precedes-last="always" name-as-sort-order="all"/>
</name>
```

adam3smith commented on isnad.csl:
```diff
@@ -0,0 +1,1387 @@
+        <multiple>Edited and translated by</multiple>
+      </term>
+      <term name="translator" form="short">trans.</term>
+      <term name="and"> - </term>
```
same here

adam3smith commented on isnad.csl:
```diff
@@ -0,0 +1,1387 @@
+            <name delimiter=" - " and="text"/>
+          </names>
+        </group>
+        <group prefix="Nşr. ">
```
I'm not sure what exactly these prefixes are, but if the style requires that you include hardcoded Turkish, it can't be a multilingual style (i.e. you'd have to set a default-locale and could remove the English locale definitions)

adam3smith commented on isnad.csl:
```diff
@@ -0,0 +1,1387 @@
+        <if type="legal_case book thesis" match="none">
+          <choose>
+            <if variable="DOI">
+              <text variable="DOI" prefix="doi:"/>
```
I'm not seeing DOIs in any of the examples in the guide. Are you sure they're supposed to be there?

adam3smith commented on isnad.csl:
```diff
@@ -0,0 +1,1387 @@
+        <if type="legal_case" match="none">
+          <choose>
+            <if variable="DOI">
+              <text variable="DOI" prefix="doi:"/>
```
ditto here

mtarakci commented:
Thank you for your interest. Your attention is fascinating. I think I can make the requested revisions. First of all, Isnad is bilingual because some journals may contain some articles in English. Secondly, I removed some suffixes before the names such as "thk." and "nşr.", and I added English term locale terms for them as "ed.". Lastly, as you rightly refer, the guide book does not contain anything about doi number. The Doi number will be added to the guide today. 
I hope that I have been able to process all the revisions correctly. Thank you for your attention and for suggesting a careful corrections.

adam3smith commented:
Thanks!
